### PR TITLE
Drop EL6, Debian 8, Ubuntu 16.04

### PR DESCRIPTION
**Pull request by ehelms**

### CODE REVIEW

**Review by ekohl — CHANGES_REQUESTED** [review]
I was looking at it and there's more EOL: Ubuntu 16.04 is EOL since April. Also Fedora < 33.

We can also drop some files in `data/`

File: metadata.json [hunk 1]
```diff
@@ -33,45 +33,39 @@
...
       ]
     },
     {
       "operatingsystem": "Fedora",
       "operatingsystemrelease": [
-        "30",
-        "31"
-      ]
+        "33",
+        "34"
```
Comment (ekohl) [inline metadata.json:57]:
```suggestion
        "34"
      ]
```

**Comment by ekohl** [comment]
Fedora 33 fails because there are no packages and Fedora 34 fails because it tries to install ntpdate which is no longer available in recent Fedora. For Fedora 34 we'll need an updated beaker: https://github.com/voxpupuli/beaker/pull/1709

**Comment by evgeni** [comment]
The title shouldn't say you're dropping 18.04, that gave me a brief shock. 16.04 is safe to drop.

**Comment by ekohl** [comment]
Turns out I was looking in the wrong place: https://github.com/voxpupuli/beaker-docker/pull/48 should fix Fedora 34.

**Comment by ehelms** [comment]
> The title shouldn't say you're dropping 18.04, that gave me a brief shock. 16.04 is safe to drop.

The commit message was right at least :)

**Comment by ekohl** [comment]
Given Fedora 33 and 34 are failing, perhaps we should roll back that change and stick to the existing values? Then we at least have green builds.

Otherwise :+1:

**Comment by ehelms** [comment]
Updated and left Fedora alone

**Review by ekohl — APPROVED** [review]

**Pull request merged**